frontend/src/scenes/onboarding/utils.tsx

```diff
@@ -56,4 +56,11 @@ export const availableOnboardingProducts: AvailableOnboardingProducts = {
         url: urls.surveys(),
         scene: Scene.Surveys,
     },
+    [ProductKey.ERROR_TRACKING]: {
+        name: 'Error Tracking',
+        icon: 'IconWarning',
+        iconColor: 'rgb(243 84 84)',
```

Comment:
Chose the same colour we have on the website but it matches surveys which is right next to it in this case